prow/build_test_update.md

```diff
@@ -20,7 +20,41 @@ bazel build //prow/cmd/hook
 bazel test //prow/plugins/lgtm:go_default_test
 ```
 
-**TODO**(cjwagner): Unify and document how to run prow components locally.
+### How to test a plugin
+
+If you are making changes to a Prow plugin you can test the new behavior by sending fake webhook events with [`phony`](/prow/cmd/phony#phony).
+
+## How to update the cluster
+
+Any modifications to Go code will require redeploying the affected binaries.
+Assuming your prow components have multiple replicas, this will result in no downtime.
+
+Update your deployment (optionally build/pushing the image) to a new image with:
+```shell
+# export PROW_REPO_OVERRIDE=gcr.io/k8s-prow  # optionally override project
+push.sh  # Build and push the current repo state.
+bump.sh --list  # Choose a recent published version
```

Comment:
This is the only part of the "How to update the cluster" section that I actually updated. The rest I just moved.

Reply:
Is this using @Katharine 's new tool?

Reply:
No, we just moved the `./bump.sh --push` functionality into the `./push.sh` script AFAICT.